views/js/runner/proxy/cache/proxy.js

```diff
@@ -145,27 +163,38 @@ define([
             var self = this;
 
             /**
-             * try to load the next item
+             * try to load the next items
              * @returns {Promise} that always resolves
              */
-            var loadNextItem = function loadNextItem(){
-                return new Promise(function(resolve){
-                    //don't run a request if not needed
-                    if(!self.isLastItem && !self.hasNextItem(uri)){
-
-                        _.delay(function(){
-                            self.request(self.configStorage.getItemActionUrl(uri, 'getNextItemData'))
-                                .then(function(response){
-                                    if(response && response.itemDefinition){
-
-                                        //store the response and start caching assets
-                                        self.itemStore.set(response.itemDefinition, response);
-                                        self.nextCalledBy[uri] = response.itemDefinition;
+            function loadNextItem() {
+                return new Promise(function (resolve) {
+                    var siblings = navigationHelper.getSiblingItems(self.testMap, uri, 'both', self.cacheAmount);
+                    var missing = _.reduce(siblings, function (list, sibling) {
+                        if (!self.hasItem(sibling.id)) {
+                            list.push(sibling.id);
+                        }
+                        return list;
+                    }, []);
 
-                                        if(response.baseUrl && response.itemData && response.itemData.assets){
-                                            assetLoader(response.baseUrl, response.itemData.assets);
-                                        }
+                    //don't run a request if not needed
+                    if (missing.length) {
```

Comment:
If you don't check if it's the last item anymore, please remove the occurrences of `isLastItem`